Libplanet/Store/BaseStore.cs

```diff
@@ -32,14 +32,18 @@ HashDigest<SHA256> hash
         /// <inheritdoc />
         public abstract IEnumerable<Address> ListAddresses(string @namespace);
 
+        /// <inheritdoc />
         public abstract void StageTransactionIds(
-            ISet<TxId> txids
+            IDictionary<TxId, bool> txids
```

Comment:
I prefer `IEnumerable<(TxId, bool)>` to `IDictionary<TxId, bool>` because we have no plans to lookup.

Reply:
I tried to change it to `IEnumerable<(TxId, bool)>`, but when `txids` is `IEnumerable` the same `TxId` can have different values.